src/main/java/fr/zcraft/quartzlib/components/i18n/translators/gettext/PluralForms.java

```diff
@@ -1,35 +1,37 @@
 /*
- * Plugin UHCReloaded : Alliances
+ * Copyright or © or Copr. Moribus (2013)
+ * Copyright or © or Copr. ProkopyL <prokopylmc@gmail.com> (2015)
+ * Copyright or © or Copr. Amaury Carrade <amaury@carrade.eu> (2016 – 2021)
+ * Copyright or © or Copr. Vlammar <valentin.jabre@gmail.com> (2019 – 2021)
  *
- * Copyright ou © ou Copr. Amaury Carrade (2016)
- * Idées et réflexions : Alexandre Prokopowicz, Amaury Carrade, "Vayan".
+ * This software is a computer program whose purpose is to allow insertion of
+ * custom images in a Minecraft world.
```

Comment:
C'est d'IoM ça non? Il faudrait un `This software is an API for Minecraft`

Reply:
Ah oui y'a eu mélange des copyrights